kasa/smart/smartdevice.py

```diff
@@ -55,7 +55,6 @@ def __init__(
         self.protocol: SmartProtocol
         self._components_raw: dict[str, Any] | None = None
         self._components: dict[str, int] = {}
-        self._state_information: dict[str, Any] = {}
```

Comment:
This was unused since converting to return `state_information` based on feature values.